### DISCUSSION TITLE
Explore: logs modify only enabled queries

### DISCUSSION THREAD

**Discussion by jmichalek132** [grafana/grafana#71694] [Data Sources]
Hi, a lot of times I have multiple queries for logs opened in explore, usually I disable all of them except one and then iterate on the one query enabled. Currently, when press the Filter for value / filter out value option on a log line, it modifies all the queries including the disabled one. I think in most cases the more desired behavior would be modifying only the enabled ones.

**Comment by ifrost** [comment]
cc @grafana/observability-logs

**Comment by matyax** [comment]
Hi @jmichalek132 ! This is something that's being actively considered for development, so might be available in the near term. We'll keep you updated.

**Reply by jmichalek132** [reply]
Great to hear that. Thank you.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82621. Feel free to continue the discussion around this there. Thank you!
